go/libraries/doltcore/sql/sqltestutil/data.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2019 Liquidata, Inc.
+// Copyright 2020 Liquidata, Inc.
```

Comment:
it seems like this is the only file left in this package, can we move it over to the `sqle` package as well?

Reply:
Yeah, I was prevented by an import cycle. I will take another pass before I'm done.